Resources/Audio/Voice/Diona/attributions.yml

```diff
@@ -0,0 +1,4 @@
+﻿- files: ["diona_scream.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "ParadiseSS13"
+  source: "https://github.com/ParadiseSS13/Paradise/commit/06bb3527c7ec597223235476103452ab518a2e58"
```

Comment:
This link doesnt go anywhere related to this sound? here should be linked the latest commit including the file at the time you took them

Reply:
The license is also likely wrong.